launch/online_async_launch.py

```diff
@@ -1,21 +1,39 @@
+from ament_index_python.packages import get_package_share_directory
+
 from launch import LaunchDescription
 from launch.actions import DeclareLaunchArgument
 from launch.substitutions import LaunchConfiguration
 from launch_ros.actions import Node
-from ament_index_python.packages import get_package_share_directory
+from nav2_common.launch import RewrittenYaml
 
 
 def generate_launch_description():
     use_sim_time = LaunchConfiguration('use_sim_time')
+    transform_timeout = LaunchConfiguration('transform_timeout')
+    params_file = get_package_share_directory('slam_toolbox') +\
+        '/config/mapper_params_online_sync.yaml'
+
+    param_substitutions = {
```

Comment:
I don't understand why this is required, its a parameter in the configuration file: https://github.com/SteveMacenski/slam_toolbox/blob/eloquent-devel/config/mapper_params_online_sync.yaml#L33. Why do you need tochange anything or make this parameter special to be substituted? I don't see that this param needs to be explicitly exposed to the launch file for any particular reason.